app/javascript/controllers/dropzone_controller.js

```diff
@@ -175,7 +177,9 @@ class DirectUploadController {
 
   start() {
     this.file.controller = this;
-    this.hiddenInput = this.createHiddenInput();
```

Comment:
Rename this function now that we're attaching *two* hidden inputs to each uploaded file.